account/account.go

```diff
@@ -32,8 +32,8 @@ import (
 // FetchAssertionFromStore retrieves an assertion from the store
 var FetchAssertionFromStore = func(modelType *asserts.AssertionType, headers []string) (asserts.Assertion, error) {
 	var user *auth.UserState
-	var authContext auth.AuthContext
-	sto := store.New(nil, authContext)
+	var storeCtx store.DeviceAndAuthContext
```

Comment:
The change to move `AuthContext` to a different modue: https://github.com/snapcore/snapd/commit/30311dbee2e11ec01f34c1c822d87e756d966e6d

This is the only API change from `snapd` I needed to add because of the new version of snapd